an3056.md

```diff
@@ -0,0 +1,21 @@
+# Open Source Development
+
+Name: Ashar Nadeem
+
+UNI: an3056
+
+GitHub: [link](https://github.com/asharnadeem)
+
+GitHub README: [link](https://github.com/asharnadeem/asharnadeem/blob/main/README.md)
```

Comment:
From the homework: 
```Note Make sure this pull request is separate from your HW0 pull request, using a different branch if necessary```

Reply:
xref: https://github.com/ColumbiaOSS/project-proposals-s2023/pull/68

Reply:
Addressed via branch switching